EpiAware/src/models.jl

```diff
@@ -1,98 +1,28 @@
 @model function log_infections(
     y_t,
-    ::Type{T} = Float64;
     epimodel::EpiModel,
-    latent_process,
+    latent_process;
+    latent_process_priors,
     transform_function = exp,
+    n_generate_ahead = 0,
```

Comment:
Is this the forecasting horizon?

Reply:
Looks like yes below, but comment would be helpful? Or a slightly more descriptive name marking it as a forecast?

Reply:
Yeah, I haven't fully fixed down the API here. I shouldn't have committed this bit yet!